### ISSUE TITLE
Implement writing to multiple targets in parallel

### ISSUE THREAD

**Issue by jhermsmeier** [resin-io/etcher#1806]
Ability to write to 15+ targets simultaneously at full speed (target ~30MB/s for SD-cards)

Depends on: #1532 #1671

**Comment by jhermsmeier** [comment]
Duplicate of #267 :facepalm:

**Comment by jviotti** [comment]
Both referenced tickets/PRs are closed, and given this is a duplicate, I guess its fine to close it.
